ui/src/main/webapp/app/table/table.html

```diff
@@ -13,216 +13,84 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
   -->
-<div ng-cloak>
-    <md-tabs md-border-bottom
-             md-dynamic-height
-             md-selected="ctrl.selectedTab" >
-        <md-tab label="Entities">
-            <md-content class="md-padding" layout="column">
-                <md-input-container>
-                    <label>Search vertices and properties</label>
-                    <input ng-model="ctrl.searchTerm" type="text">
-                </md-input-container>
+<div ng-cloak layout="column" layout-margin layout-align=" stretch"
```

Comment:
When there are no results/before you begin a query, no content at all loads. At first when I saw it it, I thought the view failed to load. Maybe a simple message telling the user that no results have been received yet would be handy.

Reply:
This also becomes a problem if you have results, but then untick edge and entity types - Then all your headers etc disappear meaning you have to navigate away and come back.